script.py

```diff
@@ -33,33 +33,76 @@ def load_text_file(filename):
             classes_name.append(currentPlace)
     return classes_name
 
+def show_predict(filename):
+    with open(filename) as f:
+        items = json.load(f)
+    print('\033[1m' + "Network prediction:")
+    item_prediction = items['prediction']
+    print('\033[0m' + "Class label:", item_prediction['label'])
+    print("Class name:", item_prediction['class_name'])
+    item_knn = items['knn parameters']
+    print('\033[1m' + "KNN parameters:")
+    print('\033[0m' + "Count neighbors:", item_knn['n_neighbors'])
+    print('\033[0m' + "Metrics:", item_knn['metrics'])
+    item_topn = items['topn']
+    print('\033[1m' + "TOP 5 predictions :")
+    for predict in item_topn:
+        print('\033[0m' + "label:", str(predict['label']),
+              "Image index:", str(predict['index']),
+              "Distance:", str(predict['distance']), sep="  ")
```

Comment:
maybe this will be better and simpler (see example)
```Python
>>> d = {'hello': 1, 'hi': 2, 'my': [1, 2, 3]}
>>> print(json.dumps(d, indent=4))
{
    "hello": 1,
    "hi": 2,
    "my": [
        1,
        2,
        3
    ]
}
```

Reply:
But your function is good too, but a little bit complicated